client/src/components/JournalPage.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import VoyagesPanel from './VoyagesPanel';
+import VoyageView from './VoyageView';
+
+function JournalPage({
+  voyages, voyage, handleVoyageClick, removeEntry, saveVoyage,
+}) {
+  return (
+    <div id="journal">
+      <VoyagesPanel voyages={voyages} handleVoyageClick={handleVoyageClick} />
+      <VoyageView voyage={voyage} removeEntry={removeEntry} saveVoyage={saveVoyage} select />
```

Comment:
Could probably set select={false} here and omit saveVoyage since we don't need a save button on the journal page.